app/helpers/react_components/mentoring/session.rb

```diff
@@ -8,7 +8,8 @@ def to_s
           "mentoring-session",
           {
             user_id: current_user.id,
-            student: {
+            partner: {
```

Comment:
Why this name change? I find the word partner a bit odd in the context of exercism.